src/content/reference/hi/p5/bezierDetail.mdx

```diff
@@ -109,8 +103,8 @@ class: p5
 params:
   - name: detail
     description: |
-      <p>number of segments to use. Defaults to 20.</p>
-    type: Number
+      <p>उपयोग किए जाने वाले खंडों की संख्या। डिफ़ॉल्ट 20।</p>
```

Comment:
Defaults to 20, can be more simplified/better explained

Reply:
Similarly across fies

Reply:
Changed all the statements involving `default` in parameters.